tests/sentry/sentry_apps/services/test_hook_service.py

```diff
@@ -43,10 +43,10 @@ def test_creates_service_hook(self) -> None:
 
     def test_expands_resource_events_to_specific_events(self) -> None:
         service_hook = self.call_create_hook(events=["issue"])
-        assert set(service_hook.events) == set(EVENT_EXPANSION["issue"])
+        assert service_hook.events == EVENT_EXPANSION["issue"]
 
     def test_expand_events(self) -> None:
-        assert expand_events(["issue"]) == set(EVENT_EXPANSION["issue"])
+        assert expand_events(["issue"]) == EVENT_EXPANSION["issue"]
```

Comment:
I know the original tests didn't have them but could we add some tests for multiple event types `i.e self.call_create_hook(events=["issue", "comment"])`. Would also be nice if we could see what the output would look like in the test 👀